mobile_verifier/src/coverage.rs

```diff
@@ -86,7 +87,11 @@ impl CoverageDaemon {
                         tracing::info!("CoverageDaemon shutting down");
                         break;
                     }
-                    Some(file) = self.coverage_objs.recv() => self.process_file(file).await?,
+                    Some(file) = self.coverage_objs.recv() => {
+			let start = Instant::now();
+			self.process_file(file).await?;
+			metrics::histogram!("coverage_object_processing_time", Instant::now() - start);
```

Comment:
is there any functional benefit to computing two different `Instant`s and then getting the difference vs. calling the `elapsed()` method on the first instant?

also, i assume the `histogram!` macro is automatically coercing the resulting duration to the necessary `f64`?